### ISSUE TITLE
`alias-model-in-controller` should support `computed.readOnly`

### ISSUE THREAD

**Issue by SaladFork** [netguru/eslint-plugin-ember#56]
This currently results in an `alias-model-in-controller` error but should probably not:
```javascript
// app/controllers/things.js
export default Ember.Controller.extend({
  things: Ember.computed.readOnly('model');
});
```

**Comment by michalsnik** [comment]
You're probably right @SaladFork, I didn't think about that scenario before but it seems like a decent practice in some cases. This should be rather straightforward to implement :)

**Comment by akshay1706** [comment]
When can we expect this to be implemented? And we would also like to know wether this case will also take into consideration the injection of other controllers model like 
```javacript
export default Ember.Controller.extend({
  thingsController: Ember.inject.controller('things'),
  things: Ember.computed.readOnly('thingsController.model');
});
```

**Comment by ryanrishi** [comment]
+1 for this. And `computed.reads` as well?

**Comment by michalsnik** [comment]
I'm going to handle `readOnly` and `reads`, but @akshay1706 I'm not sure about your example. Good practice is to alias model from corresponding route, not other controller. So in your case you probably should disable this rule per specific controller. What are your thoughts guys @netes @jbandura @kevinkucharczyk ?

**Comment by jbandura** [comment]
I agree we should probably avoid injecting controllers whenever it's possible - if it's not possible the rule should probably be disabled only in this single case.

**Comment by michalsnik** [comment]
Released in v3.2.0 🎉
